src/resolvers.ts

```diff
@@ -251,7 +251,7 @@ const resolvers = {
         createSubmission: async (_parent: unknown, args: {
             input: gqlType.CreateSubmissionInput
         }, context: UserContext): Promise<gqlType.Submission> => {
-            const isAuthorized = authorize(context.user, [Scope['write:submissions']])
+            const isAuthorized = authorize(context.user, [Scope['write:submissions']]) || authorize(context.user, [Scope['create:submissions']])
```

Comment:
double check, but I'm pretty sure authorize can take an array or Scopes instead of having to call it multiple times